src/blind.rs

```diff
@@ -137,12 +137,20 @@ impl std::error::Error for VerificationError {}
 /// Errors encountered when constructing confidential transaction outputs.
 #[derive(Debug, Clone, Copy)]
 pub enum ConfidentialTxOutError {
```

Comment:
FYI this is a breaking change, so this PR should get in before the release one.

Reply:
Sure